mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -12,12 +14,29 @@ import styles from "src/logic/userPage/UserPage.module.scss";
  */
 export const UserPage = () => {
   const {uuid} = useParams<QueryParamTypes>();
+  const [userName, setUserName] = useState<string>("");
+
+  if (!uuid) {
+    throw new Error("User is not exist");
```

Comment:
maybe if the user has not logged in, it should be redirected to another page (main, registration or 404) ?

Reply:
You are mistaking it for UserProfile, this is UserPage (like a WayPage) we are getting it by Uuid in url. But i think we should check if user with provided Uuid exist inside getUser function

Reply:
1. Uuid should be described near other page parameters and should be validated in some page wrapper. or at least hook.
2. If it is undefined or just user with the same name does not exist, user should be redirected to 404

Reply:
Moved check uuid to Routes